JavaScript/Endpoint Examples/JSON Payload/get-resource.js

```diff
@@ -1,10 +1,18 @@
 // This request demonstrates how to retrieve a resource using an ID.
 const axios = require('axios');
 
+// By default, we use the US-based API service. This is the primary endpoint for global use.
+var apiUrl = apiUrl + "";
+
+/* For GDPR compliance and enhanced performance for European users, you can switch to the EU-based service by uncommenting the URL below.
+ * For more information visit https://pdfrest.com/pricing#how-do-eu-gdpr-api-calls-work
+ */
+//var apiUrl "https://eu-api.pdfrest.com";
```

Comment:
Applies to all JavaScript samples:

Missing the actual assignment `=`
```suggestion
//var apiUrl = "https://eu-api.pdfrest.com";
```